### ISSUE TITLE
globalprotect-openconnect package installs wrong version

### ISSUE THREAD

**Issue by roworu** [NixOS/nixpkgs#148378]
### Describe the bug
globalprotect-openconnect has latest version 1.3.4 on https://search.nixos.org/packages?channel=21.11&from=0&size=50&sort=relevance&type=packages&query=globalprotect-openconnect
but when I try to install it I got version 1.2.5

### Steps To Reproduce
nix-env -iA nixos.globalprotect-openconnect

### Expected behavior
Install latest version 

### Screenshots
![image](https://user-images.githubusercontent.com/36964755/144539649-d12851ce-014e-4922-a4fa-20b4c6f570b3.png)

### Notify maintainers
@jerith666 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[roworu@nixos:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.81, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.16`
 - channels(root): `"nixos-21.11.333896.a640d8394f3"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

That my first time reporting issues, so please tell me if any additional info need to be provided.

**Comment by jerith666** [comment]
This is because of e66cf1256f4cb44bac249c332461f0cb413e5964 from https://github.com/NixOS/nixpkgs/pull/143712.  It didn't actually update the git commit hash, it just changed the hash away from base32 encoding.  :(

**Comment by jerith666** [comment]
This is the second time I've seen @r-ryantm do this (https://github.com/NixOS/nixpkgs/pull/143302 was the first).

CC @Artturin, who merged #143712, as an FYI.

**Comment by jerith666** [comment]
https://github.com/ryantm/nixpkgs-update/issues/288 is the upstream issue report for the false updates from @r-ryantm.
